src/test/java/org/kohsuke/github/GHPullRequestTest.java

```diff
@@ -119,6 +119,9 @@ public void pullRequestReviewComments() throws Exception {
         GHPullRequestReviewComment comment = comments.get(0);
         assertEquals("Sample review comment", comment.getBody());
 
+        // Assert htmlUrl is not null
+        assertNotNull(comment.getHtmlUrl());
```

Comment:
Needs to be more than non-null.

Reply:
> We need to check for more than just non-null. What value should it be?

It's actually a dynamically determined URL something along of the lines of this : `https://github.com/apache/flink/pull/5862#discussion_r184057665`

Reply:
But it is recorded so it won't change after that.

Reply:
Yep, another test upcoming  :)